src/store/reducers/rootReducer.js

```diff
@@ -28,6 +29,7 @@ const rootReducer = combineReducers({
   donMsg: donMsgReducer, // Donation messages
   comment: commentReducer, // Comments
   chat: chatReducer, // Chat
+  notifications: notificationReducer,
```

Comment:
What about a comment?

Reply:
Also consider to use inline comments.

```suggestion
  /* This is a comment */
  notifications: notificationReducer,
```